src/Content.jsx

```diff
@@ -16,6 +21,12 @@ export default class Content extends Component {
     this.props.contentDidUpdate();
   }
 
+  componentWillUnmount() {
+    if (typeof this.props.contentWillUnmount === 'function') {
```

Comment:
Remove the `typeof` check here as you provide a noop in the default props anyway